cypress/support/commands.js

```diff
@@ -1 +1,9 @@
 require('cypress-downloadfile/lib/downloadFileCommand');
+
+
+
+Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
Let's add `console.log(err?.message)` here so we at least see the exception in the console.

Reply:
Okay @ryandberger